Italian/finestra/apri.md

```diff
@@ -1,2 +1,8 @@
 
-Ti appare un paesaggio fiabesco
+Ti appare un paesaggio fiabesco e in lontananza sullo sfondo scorgi un gigantesco T-rex
+
+Come reagisci?
+
+[Prendi il telefono per fotografarlo] (../t-rex/mangiato.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.